src/utils/hibp.js

```diff
@@ -156,7 +156,7 @@ async function loadBreachesIntoApp (app) {
     app.locals.breaches = breaches
     app.locals.breachesLoadedDateTime = Date.now()
   } catch (error) {
-    throw fluentError('error-hibp-load-breaches')
+    throw FluentError('error-hibp-load-breaches')
```

Comment:
Is this also internal server error?
I think we can get rid of the concept of "FluentError" and just write out `getMessage()` when necessary. `FluentError` isn't doing too much as a wrapper imo

Hopefully when we get more of these messages standardized and translated, most if not all errors will be fluent errors

Reply:
Good idea — I removed `FluentError` in [21ef539](https://github.com/mozilla/blurts-server/pull/3010/commits/21ef539c1c3ebdbfafb41d37853f42680c42cbd9).